packages/backend/src/plugins/techInsights.ts

```diff
@@ -41,49 +41,50 @@ export default async function createPlugin(
     tokenManager: env.tokenManager,
     factRetrievers: [
       createFactRetrieverRegistration({
-        cadence: '1 1 1 * *', // Example cron, At 01:01 on day-of-month 1.
+        cadence: '1 1 1 * *', // Example cron, at 01:01 on day-of-month 1.
         factRetriever: entityOwnershipFactRetriever,
       }),
       createFactRetrieverRegistration({
-        cadence: '1 1 1 * *',
+        cadence: '* * * * *', // Example cron, every minute.
```

Comment:
Let's not change these to run every minute unless we add lifecycle configuration in as well.